src/test/java/kitchenpos/application/MenuTestFixture.java

```diff
@@ -0,0 +1,167 @@
+package kitchenpos.application;
+
+import java.math.BigDecimal;
+import java.util.Arrays;
+import java.util.List;
+import java.util.UUID;
+import kitchenpos.domain.Menu;
+import kitchenpos.domain.MenuGroup;
+import kitchenpos.domain.MenuProduct;
+import kitchenpos.domain.Product;
+
+public class MenuTestFixture {
+    private static final String ANY_MENU_NAME = "메뉴1";
+    private static final long ANY_MENU_PRICE = 2000L;
+    private static final UUID ANY_MENU_GROUP_ID = UUID.fromString("11111111-1111-1111-1111-111111111111");
+
+    private static final MenuProduct ANY_PRODUCT_WITH_QUANTITY = createMenuProduct(UUID.fromString("22222222-2222-2222-2222-222222222222"), 1);
+    static final UUID COFFEE_GROUP_ID = UUID.fromString("11111111-1111-1111-1111-111111111111");
+    static final UUID AMERICANO_PRODUCT_ID = UUID.fromString("22222222-2222-2222-2222-222222222222");
+    static final UUID LATTE_PRODUCT_ID = UUID.fromString("33333333-3333-3333-3333-333333333333");
+    static final UUID COFFEE_SET_MENU_ID = UUID.fromString("44444444-4444-4444-4444-444444444444");
+
+    static Product americano2000Won() {
```

Comment:
```suggestion
    static Product 아메리카노_2000원() {
```
개인적으론 위와 같이 Fixture는 한글로 나타내어 가독성을 높히곤 합니다 !

Reply:
다음 픽스처 작성 때 참고하겠습니다!